apps/next/src/pages/_app.tsx

```diff
@@ -115,7 +115,7 @@ export default function App({ Component, pageProps, router }: AppProps) {
       </Head>
       <Script
         src="https://cdn.jsdelivr.net/npm/magic-sdk/dist/magic.js"
-        strategy="lazyOnload"
+        strategy="afterInteractive"
```

Comment:
keeping it `afterInteractive` for now. the `loggedIn` call from magic is a bit slow and we want it to happen soon after page load. We can switch after more testing!